packages/x-charts/src/internals/plugins/corePlugins/useChartDimensions/useChartDimensions.ts

```diff
@@ -177,7 +198,7 @@ export const useChartDimensions: ChartPlugin<UseChartDimensionsSignature> = ({
       if (options?.targetElement && options?.targetElement.closest('[data-drawing-container]')) {
         return true;
       }
-      const drawingArea = selectorChartDimensionsState(store.value);
+      const drawingArea = selectorChartDrawingArea(store.value);
```

Comment:
@alexfauquette this complains that `store.value` is typed `AxisConfig[] | undefined` (simplified), but the expected input of `selectorChartDrawingArea` should `AxisConfig[]`